config.def.h

```diff
@@ -18,6 +18,15 @@ static struct key keys[] = {
     {MOD,      XK_c,   win_center, {0}},
     {MOD,      XK_f,   win_fs,     {0}},
 
+    {MOD,           XK_h,      move,  {.com = (const char*[]){"move",   "left"},  .i = 10}},
+    {MOD,           XK_j,      move,  {.com = (const char*[]){"move",   "down"},  .i = 10}},
+    {MOD,           XK_k,      move,  {.com = (const char*[]){"move",   "up"},    .i = 10}},
+    {MOD,           XK_l,      move,  {.com = (const char*[]){"move",   "right"}, .i = 10}},
+    {MOD|ShiftMask, XK_h,      move,  {.com = (const char*[]){"resize", "left"},  .i = 10}},
+    {MOD|ShiftMask, XK_j,      move,  {.com = (const char*[]){"resize", "down"},  .i = 10}},
+    {MOD|ShiftMask, XK_k,      move,  {.com = (const char*[]){"resize", "up"},    .i = 10}},
+    {MOD|ShiftMask, XK_l,      move,  {.com = (const char*[]){"resize", "right"}, .i = 10}},
```

Comment:
Paired with the other changes, this would become:

```c
    {MOD,           XK_h,      move,  {.com = (const char*[]){"m", "w"}, .i = 10}},
    {MOD,           XK_j,      move,  {.com = (const char*[]){"m", "s"}, .i = 10}},
    {MOD,           XK_k,      move,  {.com = (const char*[]){"m", "n"}, .i = 10}},
    {MOD,           XK_l,      move,  {.com = (const char*[]){"m", "e"}, .i = 10}},
    {MOD|ShiftMask, XK_h,      move,  {.com = (const char*[]){"r", "w"}, .i = 10}},
    {MOD|ShiftMask, XK_j,      move,  {.com = (const char*[]){"r", "s"}, .i = 10}},
    {MOD|ShiftMask, XK_k,      move,  {.com = (const char*[]){"r", "n"}, .i = 10}},
    {MOD|ShiftMask, XK_l,      move,  {.com = (const char*[]){"r", "e"}, .i = 10}},
```